Odata-docs/client/v7/query-options.md

```diff
@@ -11,6 +11,47 @@ ms.topic: article
 # Query options
 **Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v7.md)]
 
+There are mainly 2 ways to add Query Options to a DataServiceQuery
+1. Using AddQueryOption method.
+2. Using strongly typed C# LINQ query methods.
+
+## AddQueryOption Method
+The following example shows how to use to the `AddQueryOption` method to create a [DataServiceQuery&lt;TElement&gt;](/dotnet/api/microsoft.odata.client.dataservicequery-1) class implements the [IQueryable&lt;T&gt;](/dotnet/api/system.linq.iqueryable-1)
+
+```csharp
+DataServiceQuery peopleQuery = dsc.People
+    .AddQueryOption("$filter", "gender eq Female")
+    .AddQueryOption("$skip", "3")
+    .AddQueryOption("$orderby", "UserName desc");
+
+foreach (Person person in peopleQuery)
+{
+    Console.WriteLine($"Username: {person.UserName} First Name: {person.FirstName} Gender: {person.Gender}");
+}
+```
+
+In the case above, the URI that is generated by the Odata client includes the requested entity set with the selected query options applied.
+
+## LINQ Query Methods
+In the example below, we are creating a linq query expression that returns only Females and orders the result by UserName. We are selecting specific columns
+
+```csharp
+DefaultContainer dsc = new DefaultContainer(new Uri("https://services.odata.org/V4/(S(uvf1y321yx031rnxmcbqmlxw))/TripPinServiceRW/"));
+
+var query = dsc.People.Where(p => p.Gender == PersonGender.Female)
+    .OrderBy(p => p.UserName)
+    .Select(p => new { p.FirstName, p.LastName, p.UserName, p.Gender });
+
+foreach(var person in query)
+{
+    Console.WriteLine($"Username: {person.UserName} First Name: {person.FirstName} Gender: {person.Gender}");
+}
+```
+
+Since the [DataServiceQuery&lt;TElement&gt;](/dotnet/api/microsoft.odata.client.dataservicequery-1) class implements the [IQueryable&lt;T&gt;](/dotnet/api/system.linq.iqueryable-1) interface defined by LINQ, the OData client library is able to transform LINQ queries against entity set data into a URI that represents a query expression evaluated against a data service resource.
```

Comment:
```suggestion
Since the [DataServiceQuery&lt;TElement&gt;](/dotnet/api/microsoft.odata.client.dataservicequery-1) class implements the [IQueryable&lt;T&gt;](/dotnet/api/system.linq.iqueryable-1) interface (System.Linq), the OData client library is able to translate Linq queries against entity sets into URIs executed against a data service resource.
```

Reply:
#Resolved